DESCRIPTION

```diff
@@ -60,6 +60,8 @@ Suggests:
     MultiAssayExperiment,
     R6,
     rmarkdown (>= 2.19),
+    rvest,
```

Comment:
Nowadays I think `rvest` is just a wrapper around `xml2`. Maybe we can work directly with `xml2`? See `namespace:xml2`

```r
> rvest::read_html
function (x, encoding = "", ..., options = c("RECOVER", "NOERROR", 
    "NOBLANKS")) 
{
    UseMethod("read_html")
}
<bytecode: 0x000002a3d09722e0>
<environment: namespace:xml2>
```

<img width="658" alt="image" src="https://github.com/insightsengineering/teal/assets/133694481/f10c9be8-12a2-4248-a20d-3a829e562493">

Reply:
Nice suggestion. Let's do this before merging. `rvest` is easy to use so good for productivity.
The `rvest`/`xml2` functions are mostly going to live inside our helper functions so it won't affect the productivity of the test creation.

Reply:
Being tracked here #1162